packages/edit-site/src/components/style-book/utils.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	getBlockType,
+	getBlockTypes,
+	getBlockFromExample,
+	createBlock,
+} from '@wordpress/blocks';
+
+// Get all examples from every registered block.
+export function getExamples() {
```

Comment:
utils.js is probably not a very intuitive name for this file but what ya gunna do

Reply:
> but what ya gunna do

Call it `examples.js`, `get-examples.js` or something?

Reply:
The file contains category filters. I'm also working on getting the full list of registered categories over in https://github.com/WordPress/gutenberg/pull/65507

So maybe split up the file into `examples.js`, `categories.js` or 🤷🏻 

Naming here could be a future problem when things are fleshed out.

Reply:
> Naming here could be a future problem when things are fleshed out.

Agreed.

It might be nice before the current iterations on the Style Book are completed to split it up. Generic util files for these sorts of things tend to get a bit out of hand over time.